vm_setup_vars.yml

```diff
@@ -34,9 +34,19 @@ dns_extrahosts:
     hostnames:
       - "ns1"
   - ip: "{{ baremetal_network_cidr | nthhost(1) }}"
+    hostnames:
+     - "virthost"
+  - ip: "{{ baremetal_network_cidr_v6 | nthhost(5) }}"
+    hostnames:
+      - "api"
+  - ip: "{{ baremetal_network_cidr_v6 | nthhost(2) }}"
+    hostnames:
+      - "ns1"
+  - ip: "{{ baremetal_network_cidr_v6 | nthhost(1) }}"
```

Comment:
This change seems problematic, since you'll be creating v6 entires even on an IPv4-only deployment.

Also, `baremetal_network_cidr` ansible var is v4 or v6 depending on the deployment, so we already get v6 entries when IP_STACK=v6, we'll end up with two copies there.

Reply:
Hmm, my install was failing until I did this because the DNS query failed with `-t AAAA` succeeded afterwards

Reply:
Right, you're going to need to refactor the ansible to be more complicated than just unconditionally adding v6 entries.  If you look on line 28 above, it prefers v4 if the network is defined, so that's your problem.

Reply:
Too swamped to update this, sorry. Will have to look up the relevant ansible syntax, test, etc.